elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```
@@ -229,6 +341,28 @@ public SecurityRealm get() throws StartException {
                             if (nameRewriter == null) {
                                 nameRewriter = NameRewriter.IDENTITY_REWRITER;
                             }
+                            KeyStore keyStore = keyStoreInjector.getOptionalValue();
+                            PrivateKey privateKey = null;
+                            PublicKey publicKey = null;
+                            if (keyStore != null) {
+                                    try {
+                                        keyStoreService = getModifiableKeyStoreService(keyStoreServiceRegistry, keyStoreName);
+                                        char[] keyPassword = getKeyStorePassword((KeyStoreService) keyStoreService);
+                                        if(! keyStore.containsAlias(keyPairAlias)) {
+                                            throw new KeyStoreException();
```

We should throw an exception with an appropriate message here.